### PR TITLE
USWDS - Guidance: Create new public research recruitment page

### LINKED ISSUE #2400: USWDS-Site - UX: Publish new research recruitment page (for public)
Builds on #2274 

We need to create a new research recruitment sub-page that would be under the main Research page. The copy is finalized and ready to be added to the site.

[Recruitment page contents / copy ](https://docs.google.com/document/d/1n5soFmfRVlXUmPt2mMfz1MUy3NsDEIPSc-DdfxivVY4/edit?usp=sharing)(recruiting for the public)

On the main [Research page](https://designsystem.digital.gov/about/research/), add a "How to participate in research" section that links to this recruitment page. See copy in the [research page content doc](https://docs.google.com/document/d/1Lx-oUHboZ33aUt4XYyNU--SB_1_6Je-LHyLYHVRRCD0/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by mejiaj**
# Summary

**Create page for research recruitment (general public).** New page contains information for general public on research recruitment.

Content docs (Used internal gDocs 🔒 to make these changes):
- New page: [Usability research program recruitment page (DRAFT)](https://docs.google.com/document/d/1n5soFmfRVlXUmPt2mMfz1MUy3NsDEIPSc-DdfxivVY4/edit#heading=h.5fpfzv7ajqwr)
- Research landing page: [USWDS Research page - Content ](https://docs.google.com/document/d/1Lx-oUHboZ33aUt4XYyNU--SB_1_6Je-LHyLYHVRRCD0/edit)

**New URLs**
- `/about/research/recruitment/general-public/`
- `/research/recruitment/public` (redirect)

**Screenshot**
image

## Related issue

Closes #2400.

## Preview link

Preview links:
- [Research | How to participate in research](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-feature-recruitment-page/about/research/#how-to-participate-in-research-2)
- [Research | Recruitment for general public](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-feature-recruitment-page/about/research/recruitment/general-public/) ✨  New page

## Problem statement

Research landing needs to link to new Research recruitment page for the general public.

## Solution

Created a new section in About for research and a new page for general recruitment. This new page uses usa graphic list component to make the expectations more scannable.

This new component was created with two partials:

- `component-graphic-list.html`
- `component-graphic-list-item.html` 

This approach allows flexibility in how this component is used throughout the site. For example, we have more control over columns and how that content wraps.

**Possible limitations**
- Headers might need to be customizable to follow page order.
- Images outside of usa-icons might need to be passed.
- Current graphic list implementation has scaffolding in markdown. Editing might be difficult for non-technical folks.

## Major changes

1. Created a section for research [1f4376b].
1. Created graphic list components [e114249].

## Testing and review

- [ ] There's a new link in Research sidenav - **Recruitment for general public**.
- [ ] [How to participate in research](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-feature-recruitment-page/about/research/#how-to-participate-in-research-2) on Research landing has a link to new page for general public.
- [ ] New page for General public recruitment - [Recruitment for general public](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-feature-recruitment-page/about/research/recruitment/general-public/).

### CODE REVIEW

**Comment by mejiaj** [comment]
## CI error

The test `test_build` is showing what _seems_ to be a false error.

```bash
* At ./_site/documentation/getting-started-for-developers/index.html:2852:

  External link https://github.com/uswds/uswds#install-the-package-directly-from-github failed: https://github.com/uswds/uswds exists, but the hash 'install-the-package-directly-from-github' does not (status code 200)
```

[CI report](https://app.circleci.com/pipelines/github/uswds/uswds-site/5919/workflows/3293fdf1-00b9-4279-a0e6-2c9db4b22458/jobs/23771).

File: pages/about/research/recruitment/general-public.md
Comment (mejiaj) [inline pages/about/research/recruitment/general-public.md:2]: > [!TIP]
> The end `/` matters in permalink. Otherwise this produces inconsistent results on local vs cloud pages environment.

File: pages/about/research/overview.md
Comment (mejiaj) [inline pages/about/research/overview.md:77]: Border were added to keep it consistent with Research landing page.

File: pages/about/research/recruitment/general-public.md
Comment (mejiaj) [inline pages/about/research/recruitment/general-public.md:8]: Do we need redirects/aliases for this? For example, redirect from `research/recruitment/public/` to make it easier to type.

**Comment by annepetersen** [comment]
The title of the general public recruitment page isn't as proposed in the doc, which was: "How the public can participate in USWDS research"

**Comment by jaclinec** [comment]
I think it's looking great! My main feedback is we should tweak the "What to expect" section. I think the content under "Scheduling and information" paired with what's in "Time" is confusing - we should just put scheduling info in the "Time" section.

I propose these changes:

- Information: A member of our team will share details about that particular research effort with you so you know what to expect, and might ask you to fill out a brief consent form.
- Time: Sessions usually take 1 hour. We will provide a few date and time options to schedule with you. 
- Payment:  We generally pay $75 per session, depending on the length of the session.
- Technology needed: You typically will need a laptop, tablet, or phone, and internet access.

@annepetersen what do you think of these proposed changes?

**Comment by annepetersen** [comment]
@jaclinec  Very glad you provided the revised version: the description in the paragraph seemed perhaps more confusing to me.

But the revised version generally looks good: my only questions are if there's a better word to use than "Information" for that section, and if there's a reason you drop the "to hour and twenty" and "to $100" other than conciseness?

**Comment by jaclinec** [comment]
Hmm, not sure of a better word for "Information" - maybe "Details"?

I dropped the "hour and twenty" and the "$100" because I think we all decided that we will aim for study sessions that take no longer than 1 hour going forward. So, we likely won't pay more than $75 unless we have another reason to do so.

**Comment by annepetersen** [comment]
Ah, that's right, good reminder.

I also came up short on a better word: not sure Details is much better. I'm cool with getting this live with Information and asking our content strategists to take a look when they can. Thanks!

**Review by mahoneycm — COMMENTED** [review]
Generally looks great! I had a couple outstanding questions I left comments for.

## PR description

I noticed an incomplete sentence in the solution section:
> This new component was created with two partials that are (for now).

## Content

Also confirming @annepetersen's [comment](https://github.com/uswds/uswds-site/pull/2424#issuecomment-1888005514) that the **Sign up** section is missing this text:

> Your participation is completely voluntary, and you can decide to be taken off our list of possible participants at any time by emailing us at [uswds@gsa.gov](mailto:uswds@gsa.gov).

File: _includes/component-graphic-list-item.html
Comment (mejiaj) [reply _includes/component-graphic-list-item.html:1]: For this current implementation? Not much.

**Main reasons**
1. More control over spacing, especially in multi-column layout.
2. Opportunity to use an illustration in the future like something from [Illustrations | unDraw](https://undraw.co/illustrations) or Getty images.

**Review by annepetersen — APPROVED** [review]
Overall LGTM; may want to follow up on the redirect/alias question later

**Comment by jaclinec** [comment]
@mejiaj just giving you a heads up that we made some slight changes to the copy for the [public recruitment page](https://docs.google.com/document/d/1n5soFmfRVlXUmPt2mMfz1MUy3NsDEIPSc-DdfxivVY4/edit?usp=sharing) to be incorporated before merging. This is the final draft so no more changes are anticipated. Thanks!

File: pages/about/research/recruitment/general-public.md
Comment (mejiaj) [reply pages/about/research/recruitment/general-public.md]: Added redirect for `/research/recruitment/public` in 818cfd3.

File: pages/about/research/recruitment/general-public.md
Comment (mejiaj) [reply pages/about/research/recruitment/general-public.md]: This can be fixed once https://github.com/uswds/uswds/issues/5030 is released.

**Comment by mejiaj** [comment]
@mahoneycm, @annepetersen, @jaclinec I've added new content, addressed comments, and updated the PR description. Thanks for the suggestions. 

This is ready for another review.

**Review by jaclinec — APPROVED** [review]
Looks great - I don't have any suggested changes. Nice work!

**Comment by mejiaj** [comment]
@thisisdano ready for your review. Blocked until #2466 is merged.

**Review by mahoneycm — APPROVED** [review]
Looks great! Great work James

**Comment by annepetersen** [comment]
@thisisdano, I've reviewed this thoroughly. Would you still like to, or can we merge this in order to start recruitment from it?

**Review by thisisdano — APPROVED** [review]

**Comment by thisisdano** [comment]
Merge over pa11y crash

**Pull request merged**
